### DISCUSSION TITLE
Feature Request:  Options for System Status lines in extra status line area

### DISCUSSION THREAD

**Discussion by gitanO0** [NightscoutFoundation/xDrip#3014] [Ideas]
Would like a couple more options in the "extra status line" area (or somewhere on the main screen) showing a couple of the info lines in the g5/g6 status area of the system status page. 

Specifically would like to be able to see "Phone Service State", "Bluetooth Link" and "Brain State".
Thanks for having a look at it.

**Comment by Navid200** [comment]
I am going to convert this to a discussion.
The conversation can be had there.
